code/index.html

```diff
@@ -3,28 +3,190 @@
   <head>
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Project 3</title>
-    <!-- dont forget to add a css file and link it here! -->
+    <title>Cordonnerie</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Oswald:wght@700&family=PT+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/e056cb308d.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="shortcut icon" href="./img/favicon.ico" type="image/x-icon" />
+    <link rel="icon" href="./img/favicon.ico" type="image/x-icon" />
   </head>
+
   <body>
+    <!-- <header> -->
+    <!-- Sticky navbar -->
+    <nav class="navbar">
+      <a href="#"
+        ><img src="./img/logo.png" alt="" width="80px" height="80px"
```

Comment:
Remember to use the 'alt' attribute, it fills a great purpose and especially when it comes to accessibility. Here's some reading:
https://www.w3schools.com/tags/att_img_alt.asp